steps/set-git-credentials/1.0.0/step.yml

```diff
@@ -0,0 +1,44 @@
+title: Set Git Credentials
+summary: |
+  Sets the Git credentials used in a workflow
+description: |
+  Sets the name and email address of the Git credentials used in a workflow
+website: https://github.com/prosperllc/bitrise-step-set-git-credentials
+source_code_url: https://github.com/prosperllc/bitrise-step-set-git-credentials
+support_url: https://github.com/prosperllc/bitrise-step-set-git-credentials/issues
+published_at: 2019-12-17T12:30:47.3997059-08:00
+source:
+  git: https://github.com/prosperllc/bitrise-step-set-git-credentials.git
+  commit: 22ad52d5ddb543a96792995bc94d1ea972da5867
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
```

Comment:
Why wget is here? It does not seem to be used.

Reply:
Good question - I assume it was added by default when I generated the step template using `bitrise :step create`. I can remove it.